src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -851,13 +863,24 @@ protected void removeChildRenderTransform(View child) {
 		}
 	}
 
+	private static boolean _shouldInvalidate;
+	public static boolean getShouldInvalidate() {
+		return _shouldInvalidate;
+	}
+	public static void setShouldInvalidate(boolean shouldInvalidate) {
+		_shouldInvalidate = shouldInvalidate;
+	}
+
 	@Override
 	protected final boolean getChildStaticTransformation(View child, Transformation outTransform) {
 		Matrix renderTransform = _childrenTransformations.get(child);
 		if (renderTransform == null || renderTransform.isIdentity()) {
 			outTransform.clear();
 		} else {
 			outTransform.getMatrix().set(renderTransform);
+//			if (_shouldInvalidate) {
```

Comment:
any reason for keeping the commented code?

Reply:
There is an excellent reason... I deleted it in VS, but didn't save, then edited the file in Android Studio and lost the changes in VS

Reply:
fixed